.github/workflows/release.yml

```diff
@@ -203,9 +203,15 @@ jobs:
           SONATYPE_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
 
       - name: Release SBT plugin
+        uses: olafurpg/setup-scala@v10
         if: github.event.inputs.publish_sbt_plugin == 'Y'
         working-directory: plugins/sbt/graphql-java-codegen-sbt-plugin
-        run: sbt release with-defaults default-tag-exists-answer k
+        run: sbt ci-release
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```

Comment:
add to 
<img width="666" alt="image" src="https://github.com/kobylynskyi/graphql-java-codegen/assets/29496873/2590ff93-72e6-41e5-b4c3-a62867335362">